ACHNBrowserUI/ACHNBrowserUI/packages/Backend/Sources/Backend/environments/UserCollection.swift

```diff
@@ -55,17 +55,12 @@ public class UserCollection: ObservableObject {
                 caught += 1
             }
         }
-        return caught
-    }
-    
-    public func collectedIn(list: [Item]) -> Int {
-        var collected = 0
         for item in items {
-            if list.contains(item) {
-                collected += 1
+            if list.contains(item) && !item.name.contains("(fake)") {
+                caught += 1
             }
```

Comment:
This doesn't feel really well, cause it *could* happen, that there is a fish or bug that has the same name as an item, so it would be counted twice. First, my idea was to check if items in list are critter then iterate through critters else iterate through items but I didn't get it how to do that on a nice way... picking first element from list and using that for the check didn't felt right anyhow

Reply:
I completely agree that it could happen and I was fully aware when doing the comment. But it doesn't actually. Maybe someday an update of the game will break that, but for now, it's 100% work :)